georeference/georeference_shacl.ttl

```diff
@@ -21,76 +23,99 @@ georeference:GeoreferenceShape a sh:NodeShape ;
+            sh:message "Validation of coordinateSystemName failed!"@en ;
+            sh:name "coordinate system name"@en ;
+			sh:maxCount 1 ; 
+            sh:order 3 ;
+            sh:path georeference:coordinateSystemName ],
         [ skos:example "Ellipsodial height" ;
             sh:datatype xsd:string ;
             sh:description "Ellipsodial height or orthometric height"@en ;
             sh:in ("Ellipsodial height" "Orthometric height" "local height") ;
             sh:maxCount 1 ;
             sh:message "Validation of heightSystem failed!"@en ;
             sh:name "heightSystem"@en ;
-            sh:order 1 ;
+            sh:order 4 ;
             sh:path georeference:heightSystem ] ;
     sh:targetClass georeference:GeodeticReferenceSystem .
 
 georeference:BoundingBoxShape a sh:NodeShape ;
     sh:property [ sh:datatype xsd:float ;
             sh:message "Validation of yMin failed!"@en ;
             sh:name "yMin"@en ;
-            sh:maxCount 1 ;
+            sh:minCount 1 ;
+			sh:maxCount 1 ;
             sh:order 1 ;
             sh:path georeference:yMin ],
         [ sh:datatype xsd:float ;
             sh:message "Validation of yMax failed!"@en ;
             sh:name "yMax"@en ;
-            sh:maxCount 1 ;
+            sh:minCount 1 ;
+			sh:maxCount 1 ;
             sh:order 3 ;
             sh:path georeference:yMax ],
         [ sh:datatype xsd:float ;
             sh:message "Validation of xMin failed!"@en ;
             sh:name "xMin"@en ;
-            sh:maxCount 1 ;
+            sh:minCount 1 ;
+			sh:maxCount 1 ;
             sh:order 0 ;
             sh:path georeference:xMin ],
         [ sh:datatype xsd:float ;
             sh:message "Validation of xMax failed!"@en ;
             sh:name "xMax"@en ;
+			sh:minCount 1 ;
             sh:maxCount 1 ;
             sh:order 2 ;
             sh:path georeference:xMax ] ;
     sh:targetClass georeference:BoundingBox .
 
-georeference:Coordinate2DShape a sh:NodeShape ;
+georeference:LatLonCoordinateShape a sh:NodeShape ;
     sh:property [ sh:datatype xsd:float ;
-            sh:message "Validation of x failed!"@en ;
-            sh:name "x"@en ;
-            sh:maxCount 1 ;
+            sh:message "Validation of lat failed!"@en ;
+            sh:name "lat"@en ;
+            sh:minCount 1 ;
+			sh:maxCount 1 ;
             sh:order 0 ;
-            sh:path georeference:x ],
+            sh:path georeference:lat ],
         [ sh:datatype xsd:float ;
-            sh:message "Validation of y failed!"@en ;
-            sh:name "y"@en ;
-            sh:maxCount 1 ;
+            sh:message "Validation of lon failed!"@en ;
+            sh:name "lon"@en ;
+            sh:minCount 1 ;
+			sh:maxCount 1 ;
             sh:order 1 ;
-            sh:path georeference:y ] ;
-    sh:targetClass georeference:Coordinate2D .
+            sh:path georeference:lon ] ;
+    sh:targetClass georeference:LatLonCoordinate .
```

Comment:
Fehlt in Ontologie, siehe auch fehlschlagender Job https://github.com/GAIA-X4PLC-AAD/ontology-management-base/actions/runs/11681550013/job/32526817816?pr=102

Coordinate2D kann aus Ontologie raus.

Reply:
fixed